crates/cxx-qt-build/src/lib.rs

```diff
@@ -498,6 +498,26 @@ impl CxxQtBuilder {
             qtbuild.version().major.to_string(),
         );
 
+        for minor in 0..qtbuild.version().minor {
```

Comment:
i think you were right these need to be +1 ?

As https://doc.rust-lang.org/std/ops/struct.Range.html says that `The range start..end contains all values with start <= x < end. It is empty if start >= end.`

So this should be `0..minor+1` ?

Reply:
Ah yep indeed :) 
I am an rust expert without knowing it :)